style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  font-family: monospace;
+  background-color: lavender;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
```

Comment:
To center the text you could of used the `align-items` css property and assign it to 'center' to center it in the y-axis